packages/flutter/lib/src/widgets/primary_scroll_controller.dart

```diff
@@ -49,7 +49,7 @@ class PrimaryScrollController extends InheritedWidget {
   ///
   /// See also:
   ///
-  ///  * [ScrollView.controller], which discusses the purpose of specifying a
+  ///  * [ScrollView.getExtent], which discusses the purpose of specifying a
```

Comment:
This seems to be an accidental find and replace, and is breaking docs

Reply:
Intellij ctrl+shift+f6 caught this by accident. Fixed.